### ISSUE TITLE
--staging flag in certbot has changed to --dry-run

### ISSUE THREAD

**Issue by adasauce** [vdhpieter/docker-letsencrypt-webroot#2]
https://github.com/vdhpieter/docker-letsencrypt-webroot/blob/0d48ebd3ba5a29edd404be9b2e275438225d26cb/start.sh#L20

from: https://letsencrypt.org/docs/staging-environment/
```
The ACME URL for our ACME v2 staging environment is:

https://acme-staging-v02.api.letsencrypt.org/directory

If you’re using Certbot, you can use our staging environment with the --dry-run flag. 
```

--staging seems to just be ignored outright from certbot, no warning.

**Comment by vdhpieter** [comment]
Good catch, I'll make a patch!

**Comment by vdhpieter** [comment]
I dug a little bit deeper in the certbot documentation and found this:
https://github.com/certbot/certbot/blob/9d94c6c5ef8150fcd3c74319599f5106b9726c4c/certbot/docs/cli-help.txt#L22-L23
So it seems we need `--test-cert` but further down in the same file I found:
https://github.com/certbot/certbot/blob/9d94c6c5ef8150fcd3c74319599f5106b9726c4c/certbot/docs/cli-help.txt#L214-L218
So it seems like they still support the staging flag 🤔
They also have tests in place to test it
https://github.com/certbot/certbot/blob/9d94c6c5ef8150fcd3c74319599f5106b9726c4c/certbot/tests/cli_test.py#L287-L294
Did not run certbot code myself yet to verify because it would be a bit of a hassle

I'll keep on digging if I find something

**Comment by vdhpieter** [comment]
Ok so actually the script was completely bugged, setting the `STAGING` env will make it use production, not setting it will make it use staging 🤦 I'll make a patch and release it as a breaking change.

**Comment by vdhpieter** [comment]
Released in [v2.0.0](https://github.com/vdhpieter/docker-letsencrypt-webroot/releases/tag/v2.0.0)! Let me know if you have any more trouble/questions!
(Docker tag is building at the moment)

**Comment by adasauce** [comment]
awesome work Pieter! thanks for tracking this down.

I'll give it a shot on a fresh setup tomorrow that I'll be converting from the old kvaps one.
